river/View.zig

```diff
@@ -441,13 +442,13 @@ pub fn map(self: *Self) void {
 
     self.pending.target_opacity = self.output.root.server.config.opacity.unfocused;
 
-    log.debug(.server, "view '{}' mapped", .{self.getTitle()});
+    std.log.scoped(.server).debug("view '{}' mapped", .{self.getTitle()});
```

Comment:
let's just put this messages and the others in this file that currently have the server scope in the view scope